engine/crates/engine-v2/schema/src/input_value/display.rs

```diff
@@ -0,0 +1,98 @@
+use std::fmt::{Display, Formatter, Result, Write};
+
+use crate::{InputValue, InputValuesContext};
+
+pub struct DisplayableInpuValue<'ctx, Str, Ctx> {
```

Comment:
What format is this printing to?  JSON?  GraphQL syntax?  Arbitrary strings for a human to read?

Could use a comment to say.

Reply:
ah yes fair point, it's GraphQL syntax